doc/whats_new.rst

```diff
@@ -19,6 +19,8 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Add :class:`mne.SourceMorph` with :meth:`mne.SourceMorph.save` and :meth:`mne.SourceMorph.as_volume`, :func:`mne.read_source_morph`, :func:`mne.read_source_morph`, example plot_use_source_morph.py, by `Tommy Clausner`
```

Comment:
normally, examples are not listed here. Also, you need to add your webpage at the bottom for the link to render properly